network/src/lib.rs

```diff
@@ -45,6 +45,12 @@ use log::{error, trace};
 use std::{marker::PhantomData, sync::Arc};
 use parking_lot::Mutex;
 
+/// The head data of the parachain, stored in the relay chain.
```

Comment:
Please move that into `primitives` and consolidate with the one in `collator`

Reply:
done